src/app/components/ThemeToggle/ThemeToggle.module.scss

```diff
@@ -0,0 +1,19 @@
+@use '../../styles/variables.scss' as *;
+
+.switch {
+  position: absolute;
+  top: 2rem;
+  right: $spacing-xl;
+  width: 2rem;
+  height: 2rem;
+  background: url(/theme-dark.svg) no-repeat center center / cover;
+  transition: $transition;
+
+  &.light {
+    background-image: url(/theme-light.svg);
```

Comment:
Instead of overwriting you can use a css variable. Please take a look at the video 
https://www.youtube.com/watch?v=_2LwjfYc1x8